src/App.css

```diff
@@ -2,4 +2,231 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background: #f5f6fa;
+  font-size: 16px;
+}
+
+.panel-bg {
```

Comment:
- [ ] It would be good if the styling is the same as the one provided on Zeplin to meet the requirements. Kindly check the differences shown below and modify it 👇
- Zeplin design
<img width="614" alt="Bookstore CMS - Copy" src="https://user-images.githubusercontent.com/46851023/151613662-94e6bac0-c224-4e74-be48-013a5f86104c.png">

- Your design
<img width="960" alt="FireShot Capture 173 - Book Store - localhost" src="https://user-images.githubusercontent.com/46851023/151613896-4098f3d0-b6ce-458b-bcab-dfe369127d7d.png">

The differences are marked with black colour, kindly use similar spacing and add the missing elements.